content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md

```diff
@@ -65,21 +60,13 @@ Follow these steps:
 
 1. Connect a [compatible board](https://support.arduino.cc/hc/en-us/articles/360014779899#chromebook) to one of the USB ports on your Chromebook.
 
-1. In the board selector menu, choose **Select Other Board & Port**
-
-   ![The Boards Dropdown in the Cloud Editor.](img/web-editor-boards-dropdown.png)
-
-1. Pick the specific board you're using and click **OK**.
-
-1. Click the **Upload** button ![Upload button](img/upload_button.png) in the editor.
-
-1. That should open a **Authorise USB connection** window. Click **Select port**.
+1. In the board selector menu, click on the **Detect Device** button.
 
-   ![Authorise USB connection](img/Authorise-USB-connection.png)
+   ![The Detect Device button.](img/detect-device-button.png)
 
-1. A **ChromeOS** window should appear. Select the desired port and click **Connect**.
+1. Select the serial port for your board board you're using and click **Connect**. If you're unsure which serial port to use, try disconnecting and reconnecting your device. The serial port for your board will disappear and reappear in the list.
 
-   ![ChromeOS window](img/usbpromptchrome.png)
```

Comment:
`img/usbpromptchrome.png` is now unused so the file should be removed from the repository.

Reply:
Resolved by 68d8538bd1a45060e958b183f26aa132304e0b89.